### DISCUSSION TITLE
Is devcontainers/ci expected to mount repo while building the image?

### DISCUSSION THREAD

**Discussion by lmilbaum** [devcontainers/community#44] [Q&A] [answered]
I would like to use the devcontainer settings in CI. The expectation is the action to mount the repo folder while building the image and running the container when performing the `runCmd`. But, this is not what is happening in my case. 
I've added a `COPY . . ` instruction but that doesn't copy the repo root folder. It copies the content of the .devcontainer folder. 
What am I missing?

**Comment by Chuxel** [comment, marked as answer]
When building an image, the context is relative to the location of `devcontainer.json`. Since this is typically in the `.devcontainer` folder, you'll want to set `"context": ".."` to be able to pull in content from the repo root via COPY.

```
"build": {
    "dockerfile": "Dockerfile",
    "context": ".."
}
```

**Reply by lmilbaum** [reply]
Thank you for your feedback. I've came to same result as you suggested.
